ebooks.py

```diff
@@ -233,6 +235,7 @@ def grab_toots(api, account_id=None,max_id=None):
                     sys.exit()
 
             if not DEBUG:
+                ebook_status=filter_status(ebook_status)
```

Comment:
this filtering has happened when all of the statuses are grabbed from Twitter or Mastodon, so I don't think this will be necessary here.